static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -131,13 +177,33 @@ function DefaultGroupEventDetailsContent({
         />
       </StyledDataSection>
       {event.userReport && (
-        <EventDataSection title={t('User Feedback')} type="user-feedback">
-          <EventUserFeedback
-            report={event.userReport}
-            orgSlug={organization.slug}
-            issueId={group.id}
-            showEventLink={false}
-          />
+        <EventDataSection
+          title={t('User Feedback')}
+          type="user-feedback"
+          actions={
+            // Button is disabled if promptLoading or promptError.
+            <ErrorBoundary mini>
+              <HideUserFeedbackButton
+                isLoading={promptLoading}
+                isError={promptError || isPromptDismissed === undefined}
+                isHidden={isPromptDismissed ?? false}
+                showFeedback={showPrompt}
+                hideFeedback={dismissPrompt}
+              />
```

Comment:
would it be better to just move the button code here directly? it's not that much more code, and you'll have less prop passing

Reply:
Truu